sources/schema/club.js

```diff
@@ -79,13 +78,13 @@ const queries = `
         domain:String
     ):[Club]
     clubByName(name:String!):Club
-    clubById(id:String!):Club
-    deleteClub(id:ID!):ResponseResult
+    clubById(id:String!):Club    
 `;
 
 const mutations = `
     addClub(club:ClubInputType):ClubResult,
     updateClub(id:ID!,club:ClubInputType):ClubResult,    
+    deleteClub(id:ID!):ResponseResult
```

Comment:
If a club is deleted, is the clubID removed from the user's collection? as club member thingy?

Reply:
it deletes the AccessLevel objects, but not the user's clubAccess object yet. I'll handle that, thanks for pointing it out